deploy/crds/infra.watch_servicetelemetrys_crd.yaml

```diff
@@ -167,49 +167,67 @@ spec:
                     description: Events related backend configuration.
                     properties:
                       elasticsearch:
-                        description: Events storage backend ElasticSearch
+                        description: Events storage backend Elasticsearch
                         properties:
                           enabled:
-                            description: Enable ElasticSearch as a storage backend for events
+                            description: Enable Elasticsearch as a storage backend for events
+                            type: boolean
+                          extHostUrl:
```

Comment:
I see your points and logic of how you got here :)

I'm going to think about it some and will get back to you.

Reply:
Looking at the CRD there are two suggestions I came up with. Simply noting as a discussion point.

I was looking at the `storage` section and thinking about adding a new strategy called `- external` and then adding an `external` section (like we have a `persistent` section). Thinking some more about that I thought it might be awkward and not very intuitive.

I then thought about the `external` subsection again, and was wondering if maybe the naming is the awkward bit? What about instead of external we use a section called `forwarding`? It doesn't really change anything about the syntax, but would give an opportunity to add another `description` section that would explain what the purpose of the configurations would be.

There is also the option where we can remove the old syntax from the CRD, but still allow it to be passed in and consumed for the lifecycle of STF 1.5 so that we don't totally break any existing systems. That would at least help clean up the CRD itself and the UI interface. It might even be something to consider for the first round where the old CRD parameters are removed, but still accepted by the Ansible roles.

Ideally we can accept the old syntax but then have the code basically accept the existing managed ES that we are deploying as the new forwarded endpoint without requiring an administrator to remove it, but at which point we'd drop all management responsibility and simply have the `forwarding` (`external`, `ext<param>`, ...) section be populated from the old params.

Hope that makes sense. Maybe when you're back we can all get together during one of backlog grooming meetings or otherwise and discuss?

Reply:
Implementing this as a `forwarding` section